Vagrantfile

```diff
@@ -102,8 +101,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     configure_ansible_provisioning(vm_config)
   end
 
+  config.vm.define("el9", primary: true) do |vm_config|
+    vm_config.vm.box = "generic/centos9s"
+    vm_config.vm.host_name = "candlepin-el9.example.com"
+
+    # Uncomment these lines for forward the Candlepin standard dev ports to this guest.
```

Comment:
Hrmm... on second thought, maybe those commented lines shouldn't get moved -- they're intended to be used only in a single VM at a time. Though it could still be removed from all but one of those blocks and explained a bit more, perhaps.

Reply:
Done.